### ISSUE TITLE
[KEY-257] Create a better onboarding flow

### ISSUE THREAD

**Issue by chronark** [unkeyed/unkey#528]
1. the user sets up clerk + planetscale 
2. npx @unkey/cli bootstrap …
   \- pushes schema to planetscale
   \- creates workspace, api and keyAuth
   \- sets correct .env variables
3. pnpm dev
   \- runs the nextjs web app on :3000
   \- runs the cloudflare api on :8787

<sub>From [SyncLinear.com](https://synclinear.com) | [KEY-257](https://linear.app/unkey/issue/KEY-257/create-a-better-onboarding-flow)</sub>

**Comment by spiritanand** [comment]
@chronark I would be up to take this.

**Comment by bipulpoudel** [comment]
@chronark , Is it okay, if I pick it up?

**Comment by chronark** [comment]
step 3 is not quite possible yet, we're blocked on #458 
@perkinsjr has done a prototype for the cli wizard, I'll defer to him

**Comment by perkinsjr** [comment]
Yeah I have plans to finish out the prototype this weekend. I assigned this to myself

**Comment by ErrolFernandes** [comment]
> step 3 is not quite possible yet, we're blocked on #458 @perkinsjr has done a prototype for the cli wizard, I'll defer to him

Since #458 is merged this is possible now?

**Comment by perkinsjr** [comment]
> > step 3 is not quite possible yet, we're blocked on #458 @perkinsjr has done a prototype for the cli wizard, I'll defer to him
> 
> Since #458 is merged this is possible now?

Yes this is possible today. I have this on my todos to finish the CLI for the onboarding contributors, although it's low priority.

**Comment by ErrolFernandes** [comment]
> > > step 3 is not quite possible yet, we're blocked on #458 @perkinsjr has done a prototype for the cli wizard, I'll defer to him
> > 
> > 
> > Since #458 is merged this is possible now?
> 
> Yes this is possible today. I have this on my todos to finish the CLI for the onboarding contributors, although it's low priority.

Will you also add the docs for this?

**Comment by perkinsjr** [comment]
This is done in the contribution section of our docs.
